stories/setup/config.tsx

```diff
@@ -22,7 +22,7 @@ import './styles.scss';
 configureApplication();
 
 // Automatically import all files ending in *.stories.tsx
-const req = require.context('../', true, /.stories.tsx$/);
+const req = require.context('../', true, '../', /stories.tsx$/);
```

Comment:
[`require.context()`](https://github.com/webpack/docs/wiki/context#requirecontext) only takes 3 arguments and you'll need a regular expression in the third one so take a look at this again.